solutions/community-deploy/CORTX-Deployment.md

```diff
@@ -1,84 +1,106 @@
+### You must have Git installed on your system.
+You can use following command for RedHat family OS to install the Git
 
 ```
-      yum install git -y
+# yum install git -y
 ```
+
 ### SELinux should be disabled
-   - Use below command to check status of SELinux.
+
+-  Use the following command to check status of SELinux.
 ```
-      sestatus
+# sestatus
 ```
-   - If SELinux is enable then you can run below command to disabled SELinux.
+-  If SELinux is enable, run the following command to disable the SELinux.
 
 ```
-      sed -i 's/SELINUX=enforcing/SELINUX=disabled/' /etc/selinux/config && setenforce 0
+# sed -i 's/SELINUX=enforcing/SELINUX=disabled/' /etc/selinux/config && setenforce 0
 ```
 
-   - Once above command ran successfully then reboot your system.
+-  Once above command runs successfully, reboot your system.
 
 ```
-      reboot
+# reboot
 ```   
+
 ### Note
- 1. All nodes should be reachable over SSH with root user
- 2. You also deploy CORTX on AWS EC2 instance. Please follow [CORTX Deployment on AWS](https://github.com/Seagate/cortx-re/blob/main/solutions/community-deploy/cloud/AWS/README.md)
+ 1. All the nodes should be reachable over SSH with root user.
+ 2. You can deploy CORTX on AWS EC2 instance also. Please follow [CORTX Deployment on AWS](https://github.com/Seagate/cortx-re/blob/main/solutions/community-deploy/cloud/AWS/README.md)
 
 ## Install K8s cluster
+**To install the K8s cluster, run the following commands:**
 
-   - Clone cortx-re repository and change directory to `cortx-re/solutions/kubernetes`
+-  Clone cortx-re repository and change directory to `cortx-re/solutions/kubernetes`
 ```
-      git clone https://github.com/Seagate/cortx-re 
-      cd $PWD/cortx-re/solutions/kubernetes
+# git clone https://github.com/Seagate/cortx-re && cd $PWD/cortx-re/solutions/kubernetes
 ```
-   - Create hosts file by below command, change root user password in below command.
+
+-  To create the host file and change the root user password with your system password, run the following command
 ```
-      echo "hostname=$(hostname),user=root,pass=rootuserpassword" > hosts && cat hosts
+# echo "hostname=$(hostname),user=root,pass=rootuserpassword" > hosts && cat hosts
 ```
-   - Execute `cluster-setup.sh` to setup K8s cluster on your VM. If you wanted to run pods on to master node as well then we need to give first input parameter for `cluster-setup.sh` script true else false. If you have single node cluster then pass input parameter should be true.
+
+-  Execute `cluster-setup.sh` to setup K8s cluster on your VM.  
+-  If you want to run pods on to the master/primary node as well, pass the first input parameter for `cluster-setup.sh` script as true.  
+-  If you have single node cluster, pass the input parameter as true.
+
 ```
-      ./cluster-setup.sh true
+# ./cluster-setup.sh true
 ```
 
-## Deploy Cortx Stack on K8s Cluster
+## Deploy the CORTX Stack on K8s Cluster
```

Comment:
We have the heading `Deploy the CORTX Stack on K8s cluster` at the top. So this should be only `Deploy CORTX Stack`